i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/integrate-logto/fragments/_configure-post-sign-out-redirect-uri.mdx

```diff
@@ -0,0 +1,8 @@
+<p>
+  让我们在这里切换到「管理控制台」的应用详情页。添加一个值为 {}
```

Comment:
forget to replace `{}`?

Reply:
This `{}` add a space before the config value:
<img width="967" alt="image" src="https://user-images.githubusercontent.com/10806653/187585753-e42566ad-d492-4d78-801d-1229752d5938.png">